### ISSUE TITLE
FGDC XSLT update

### ISSUE THREAD

**Issue by jbrown-xentity** [GSA/data.gov#1520]
## User Story

As a member of FGDC, I want to confirm the newly designed XSLT transform file for FGDC2ISO service will work appropriately so that we have the best transform service possible.

## Acceptance Criteria
- [x] Various harvest sources are tested on staging using the new XSLT [transform file](https://github.com/GSA/datagov-deploy/files/4307367/csdgm_iso_transform.zip)
- [ ] Any/all harvest errors are tracked and verified as expected with Image Matters team.
- [x] New ticket is made for production release of XSLT

## Details
Depends on #865. Should then build in sandbox and provide targeted harvest results to FGDC team

**Comment by FuhuXia** [comment]
We have done [a list of modifications](https://github.com/GSA/ckanext-geodatagov/commits/f28f1751d23eac973d96394a2485ccfbd847135b/conversiontool/fgdc2iso/fgdcrse2iso19115-2.xslt) to previous XSLT file. Each mod need to be inspected and make sure original issue is addressed.

**Comment by FuhuXia** [comment]
Went through a list of FGDC harvest sources. All are harvested twice using existing fgdcrse2iso19115-2.xslt and new csdgm2iso19139-20200109.xslt. Harvesting job results look identical. 

Will verify more and compare the transformed metadata between to xslt files.

**Comment by FuhuXia** [comment]
Did a comparison between two ISO19139 XML files generated by the two XSLT. The content of the two files are mostly matching, except when it comes to `schemaLocation` values; existing one uses `http://www.ngdc.noaa.gov`, new one uses `http://schemas.opengis.net`. I think that is expected.

Will open a new ticket to use the install new XSLT file to our fgdc2iso repo.
